mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -334,3 +345,22 @@ properties:
         name: 'expirationTime'
         description: |
           The expiration time of the schedule. The timestamp is an RFC3339 string.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'diskConsistencyGroupPolicy'
+    min_version: 'beta'
+    conflicts:
+      - 'snapshot_schedule_policy'
+      - 'group_placement_policy'
+      - 'instance_schedule_policy'
+    description: |
+      Replication consistency group for asynchronous disk replication.
+    send_empty_value: true
+    properties: 
+      - !ruby/object:Api::Type::Boolean
+        name: 'enabled'
+        default_value: true
```

Comment:
Let's avoid having a default_value for this field. Can we make this field required instead? If the value is returned from the api by default we can add `default_from_api`.